.mockery.yaml

```diff
@@ -26,14 +26,18 @@ packages:
             CCIPReader:
             PriceReader:
             RMNHome:
-    github.com/smartcontractkit/chainlink-ccip/pkg/peergroup:
-        interfaces:
-            PeerGroupFactory:
-            PeerGroup:
     github.com/smartcontractkit/chainlink-ccip/pkg/contractreader:
         interfaces:
             Extended:
             ContractReaderFacade:
     github.com/smartcontractkit/chainlink-ccip/pkg/types/ccipocr3:
         interfaces:
             ExecutePluginCodec:
+    github.com/smartcontractkit/libocr/networking:
+        interfaces:
+            PeerGroupFactory:
+                config:
+                    dir: mocks/libocr_networking/
```

Comment:
Needed otherwise the dir would be `github.com/smartcontrackit/libocr/networking`